src/it/basic-dependencyManagement/pom.xml

```diff
@@ -1,31 +1,31 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-  <modelVersion>4.0.0</modelVersion>
-  <groupId>localhost</groupId>
-  <artifactId>basic-single-dependency</artifactId>
-  <version>1.0</version>
-  <packaging>pom</packaging>
-  <name>libyear-report</name>
-
-  <description>Testing that versions from dependencyManagement are
-		handled correctly</description>
-  <url>http://localhost/</url>
-
-  <dependencyManagement>
```

Comment:
There's a lot of changes in this MR that shouldn't be here. If I run 

```
git checkout cominto/main
mvn spotless:apply
```

then I get a large diff. Does the same happen for you?

Reply:
I haven't seen Spotless fail to apply formatting for one person but not another before. I'll try and do some research, but to unblock this, are you OK with me cherry-picking this commit and running the formatting myself? I can then raise it as a different PR. Alternatively I can raise a PR into your repo and then this one should get updated when it's merged.